index.html

```diff
@@ -8,33 +8,179 @@
+            <p class="welcome-message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a class="heading-1">LET’S CONNECT</a>
+            <ul class="icon-bar">
+                <li>
+                    <img src="images/twitter.png" alt="">
+                </li>
+                <li>
+                    <img src="images/Linkedin icon.png" alt="">
+                </li>
+                <li>
+                    <img src="images/m.png" alt="">
+                </li>
+                <li>
+                    <img src="images/cat.png" alt="">
+                </li>
+                <li>
+                    <img src="images/hand.png" alt="">
+                </li>
+            </ul>
+        </section>
+       </header>
+       <main>
+        <section class="work" >
+            <div class="tonic-1">
+                <img src="images/tonic.png" alt="">
+                <p class="tonic">Tonic</p>
+                <div class="canopy">
+                    <p>Canopy</p>
+                    <ul class="dev">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>
+                        <button class="html">html</button>
+                    </li>
+                    <li>
+                        <button class="css">css</button>
+                    </li>
+                    <li >
+                        <button class="javascript">javascript</button>
+                    </li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+            <div class="multi-post-stories-container">
+                <img src="images/multi-post-stories.png" alt="">
+                <p class="tonic">Multi-post stories</p>
+                <div class="canopy">
+                    <p>Canopy</p>
+                    <ul class="dev">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>
+                        <button class="html">html</button>
+                    </li>
+                    <li>
+                        <button class="css">css</button>
+                    </li>
+                    <li >
+                        <button class="javascript">javascript</button>
+                    </li>
+                </ul>
+                <button class="btn">See Project</button>
```

Comment:
<img width="679" alt="1" src="https://user-images.githubusercontent.com/77843050/219652333-fe7de904-2489-482c-bed8-d4f057d13263.png">

- [ ] Please make sure you follow the design guidelines, seeing the 2 images, there are still some few sections of improvement, it is required to use the same text properties (`font size, font weight, color, etc...`) that are used in the Figma was given design, so please make sure you adjust the padding between texts, colors, and fonts too`
- [ ] Your technology buttons are having a different background I think, please update that too.